src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -731,15 +731,23 @@ public void PreAuthenticate_SetAndGetBoolean_ValuesMatch(Uri remoteServer)
             Assert.False(request.PreAuthenticate);
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public void PreAuthenticate_SetAndGetBooleanResponse_ValuesMatch(Uri remoteServer)
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task PreAuthenticate_SetAndGetBooleanResponse_ValuesMatch(bool useSsl)
         {
-            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
-            request.PreAuthenticate = true;
-            using (var response = (HttpWebResponse)request.GetResponse())
+            var options = new LoopbackServer.Options { UseSsl = useSsl };
```

Comment:
This test is sort of strange. I can't remember exactly what it is validating other than doing a 'get' of the .PreAuthenticate property after doing a 'set'.  This property doesn't have anything to do with TLS/SSL. It is about pre-authenticating for HTTP Auth such as Basic/Digest, etc.

I'm not even sure why it needs to send a request to a server at all if all it is doing is verifying that setting a property works. It's not really doing any end-to-end validation.

So, at a minimum you don't need to set the 'UseSsl'. And thus you don't need the .ServerCertificateValidationCallback either.

@stephentoub Do you remember the origin of this test?

Reply:
According to git history, the origin of the test is @ViktorHofer added it in https://github.com/dotnet/corefx/pull/19118, presumably as part of increasing code coverage :smile:

Reply:
_hiding under a rock_

Reply:
Fixed